keyboards/redox/keymaps/eightbitraptor/keymap.c

```diff
@@ -0,0 +1,89 @@
+#include QMK_KEYBOARD_H
+
+// Each layer gets a name for readability, which is then used in the keymap matrix below.
+// The underscores don't mean anything - you can have a layer called STUFF or any other name.
+// Layer names don't all need to be of the same length, obviously, and you can also skip them
+// entirely and just use numbers.
+#define _QWERTY 0
+#define _SYMB 1
+#define _NAV 2
+#define _ADJUST 3
```

Comment:
Using an `enum` statement here will make your layers easier to manage if you change them in the future. Lines 7-10 could be replaced with:

```c
enum layer_names {
    _QWERTY,
    _SYMB,
    _NAV,
    _ADJUST,
};
```

This will compile to the same result you have now, but you won't have to manually update the numbers if you change your layers (adding or reordering).

Reply:
^ This.

This isn't strictly required, but ... as somebody that has used both, the enum is definitely easier to manage, long term.

Reply:
Thanks folks. I'll get that changed straight away.